homeassistant/components/intesishome/climate.py

```diff
@@ -375,7 +375,7 @@ async def async_update_callback(self, device_id=None):
             )
             # Schedule reconnection
             async_call_later(
-                self.hass, reconnect_minutes * 60, self._controller.connect()
+                self.hass, reconnect_minutes * 60, self._controller.connect
```

Comment:
This won't work, because `async_call_later` calls the method with a timestamp. Instead, create a new method

```python
async def try_connect(_now):
    await self._controller.connect()
```

And pass that.